web/app/configs/lang.json

```diff
@@ -376,5 +376,68 @@
         "TR": "Turkiet",
         "UA": "Ukraina"
     }
+  },
+  "de": {
+    "wind": "Wind",
+    "solar": "Solar",
+    "hydro": "Wasser",
+    "hydro storage": "Pumpspeicher",
+    "biomass": "Biomasse",
+    "nuclear": "Atomkraft",
+    "geothermal": "Geothermie",
+    "gas": "Gas",
+    "coal": "Kohle",
+    "oil": "Öl",
+    "unknown": "unbekannt",
+    "exportto": "des Exports zu",
+    "importfrom": "des Imports von",
+    "productionimport": "Produktion/Import",
+    "consumption": "Verbrauch",
+    "electricityfrom": "Elektrizität kommt von",
+    "electricityto": "Elektrizität wird exportiert nach",
+    "electricitystored": "Elektrizität wird gespeichert mit",
```

Comment:
`Ist` works only in the past tense here as in `ist exportiert worden`, `wird exportiert` is present tense. ;-)